index.php

```diff
@@ -1,18 +1,20 @@
 <?php
 require('functions.php');
+$db = getDatabase();
+$cartoonArray = queryDatabase($db);
 ?>
 
 <!DOCTYPE html>
```

Comment:
Unless mike doesn't mind, I don't think it's needed?

Reply:
Yeah I wasn't sure if needed.